airbyte-webapp/src/utils/useTranslateDataType.ts

```diff
@@ -3,19 +3,37 @@ import { useIntl } from "react-intl";
 
 export interface AirbyteConnectorData {
   type: string;
+  $ref?: string;
```

Comment:
Any tests need to be updated or added for this change?

Reply:
The UI change happened in this PR because it was breaking some integration tests.